src/extension.ts

```diff
@@ -1,10 +1,30 @@
 import * as vscode from 'vscode';
+import { LocalHistoryProvider } from './local-history';
 
 export function activate(context: vscode.ExtensionContext) {
+    const provider = new LocalHistoryProvider();
+
+
```

Comment:
[minor] keep a single-line between statements. 
Do you mind adding the following to eslint to prevent multiple empty lines?
https://github.com/eclipse-theia/theia/blob/cb3db2a1191b350bc46784d8e699e4f15a7647d8/configs/errors.eslintrc.json#L47-L52